src/main/resources/icons/reset.svg

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Nice, where's that taken from?

Reply:
It's a combination of two icons from IntelliJ  :)

Reply:
haha okay :rofl: